Lib/AspNet/Microsoft.WindowsAzure.Storage/project.json

```diff
@@ -1,91 +1,74 @@
 ﻿{
-    "version": "7.0.1.0",
+    "version": "7.1.1.0",
     "authors": [ "Microsoft Corporation" ],
     "description": "Azure Storage SDK for CoreCLR Preview",
     "dependencies": {
         "Microsoft.Data.OData": "5.6.4",
```

Comment:
Hey @GuardRex, I understand that you want a GA release. This is not the release you are looking for :wink:. It is a release vehicle we are using to add preview support for Core RTM. Like any preview, we welcome any comments you have but also like any preview there should be no expectation that things are GA quality at this point.

I guess you are just ranting, but I'm really not sure whether you're actually blocked by OData or JSON.NET. If so, please do open an issue; my understanding is that these are not blocking issues.

I think you are largely picking on a bunch of minor things. The changelog is right there at the root of the repo and also included with the text of every release. An announcement repo was determined to be overkill for this project given the other announcement channels available.

However, I will say that you are right about public development. We are making improvements in this area. We highly value the input that the community provides and we'll do what we can to make sure our processes reflect that. (But please try to be polite with your criticism... :laughing:)